Exiled.API/Features/Player.cs

```diff
@@ -2386,13 +2387,6 @@ public Item AddItem(ItemBase itemBase, Item item = null)
                     acquisitionConfirmationTrigger.AcquisitionAlreadyReceived = true;
                 }
 
-                // Dont care, didnt ask, ratio
-                Timing.CallDelayed(0.02f, () =>
-                {
-                    if (item.Type is ItemType.SCP330 && item.Base != null)
-                        ((Scp330)item).Base.ServerRefreshBag();
```

Comment:
This is necessary for the client to get the contents of the bag properly.

Reply:
i will like to request someone to test it at least to be 100% sure